src/main/resources/emissary/util/ImageIOClass.cfg

```
@@ -0,0 +1,35 @@
+# Image IO Classes that seem to need some help getting loaded
+IMAGEIO_CLASS = "com.github.com.jaiimageio.impl.plugins.bmp.BMPImageReaderSpi"
+IMAGEIO_CLASS = "com.github.com.jaiimageio.impl.plugins.bmp.BMPImageWriterSpi"
+IMAGEIO_CLASS = "com.sun.imageio.plugins.gif.GIFImageWriterSpi"
```

We seem to have "com.sun.imageio.plugins.gif.GIFImageWriterSpi" in both the original list (line 4) and the SURPLUS list (line 29).  That will cause it to be registered and then deregistered in short succession.  I believe line 4 should be GIFImageReaderSpi.

```suggestion
IMAGEIO_CLASS = "com.sun.imageio.plugins.gif.GIFImageReaderSpi"
```

---

Thanks for catching that, it was a typo